src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj

```diff
@@ -31,22 +22,28 @@
     <PackageReference Update="Microsoft.NETCore.App" PrivateAssets="All" />
   </ItemGroup>
 
-  <Target Name="BuildX64" AfterTargets="Build" Condition="'$(TargetFramework)'=='net461' AND '$(PlatformTarget)'!='x64'">
+  <Target Name="BuildX86" AfterTargets="Build" Condition="'$(TargetFramework)'=='net461' AND '$(PlatformTarget)'!='x86'">
```

Comment:
A couple of fixes in this change - inverting the change so that it matches what EF does (https://github.com/aspnet/EntityFrameworkCore/blob/dev/src/ef/ef.csproj#L43). It makes the PlatformTarget checks consistent with the produced exe flavor.